snowdialect.py

```diff
@@ -572,18 +572,49 @@ def get_schema_names(self, connection, **kw):
 
         return [self.normalize_name(row[1]) for row in cursor]
 
-    def get_table_comment(self, connection, table_name, schema=None, **kw):
+    def _get_table_comment(self, connection, table_name, schema=None, **kw):
         """
-        Returns comment of table in a dictionary as described by SQLAlchemy spec
+        Returns comment of table in a dictionary as described by SQLAlchemy spec.
         """
         sql_command = "SHOW /* sqlalchemy:get_table_comment */ " \
                       "TABLES LIKE '{0}'{1}".format(
                                         table_name,
                                         (' IN SCHEMA {0}'.format(self.normalize_name(schema))) if schema else ''
                                     )
         cursor = connection.execute(sql_command)
-        ans = cursor.fetchone()
-        return {'text': ans['comment'] if ans['comment'] else None}
+        result = cursor.fetchone()
+        return result['comment']
```

Comment:
This needs to be changed back.
There was another bug where the empty string comment would trick alembic into thinking that there's a comment.
We need to check `result['comment'] == ''` and then instead return `None`.

Reply:
@keller00 please note that I'm checking this value in `get_table_comment` now, rather than repeating the logic in both helper functions, so this should be taken care of.